core/src/org/apache/cloudstack/storage/command/AttachAnswer.java

```diff
@@ -34,6 +37,12 @@ public AttachAnswer(DiskTO disk) {
         setDisk(disk);
     }
 
+    public AttachAnswer(DiskTO disk, Map<String, String> diskDetails) {
```

Comment:
@GabrielBrascher Thanks. I started some small feature to be able to save/retrieve disk details, it is likely I dropped it but forgot to remove remove this -- I'll check and remove this.